src/main/java/us/model/Payment.java

```diff
@@ -15,7 +15,52 @@
     })
     private Participation participation;
     private String name;
+    private String ssoId;
```

Comment:
i don't know why `ssoId` field is nee.
how about include things that associated with `ssoId` to `Participation` model?